CONTRIBUTING.md

```diff
@@ -24,7 +19,8 @@ Making a Contribution
 1. Write a test which shows that the bug was fixed or that the feature works as expected.
 1. Send a pull request (PR) and work with us until it gets merged and published. Contributions may need some modifications, so a few rounds of review and fixing may be necessary. We will include your name in the ChangeLog :)
 1. For quick merging, the contribution should be short, and concentrated on a single feature or topic. The larger the contribution is, the longer it would take to review it and merge it.
-1. Mbed TLS is released under the Apache license, and as such, all the added files should include the Apache license header.
+1. Ensure that all new files include the [Apache-2.0](https://spdx.org/licenses/Apache-2.0.html) standard license header.
+1. Ensure that each commit has at least one `Signed-off-by:` line, using your real name and email address. The names in the `Signed-off-by:` and `Author:` lines must match. If anyone else contributes to the commit, they must also add their own `Signed-off-by:` line. By adding this line the contributor(s) certify that the contribution is made under the terms of the [Developer Certificate of Origin](dco.txt).
```

Comment:
I guess this is quite common these days, but this isn't how the team have been working so far. I don't object to us adopting this - but if we do, will this also apply to the internal team, (I presume) and not just external contributors, so should this apply for all work from this point onwards? How are we rolling this out?

Reply:
We're discussing internally. Arm owns the copyright, so there's no legal requirement for Arm employees acting in their professional capacity to make a legal statement through a Signed-off-by line. We intend to make it a CI-enforced requirement for everyone eventually, but for the time being this is only a hard requirement for non-Arm-employees who aren't covered by a CLA.

Reply:
Regarding the change here, I copied this from another project but I now think this is too restrictive. It's enough for the committer to have provided a signed-off-by line; this doesn't have to be the author. I will update.